wp-content/mu-plugins/premium-plugins.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
 add_action( 'init', function() {
-	if ( ! class_exists( 'ElementorPro\Plugin' ) ) {
+	if ( file_exists( ABSPATH . '/wp-content/plugins/elementor-pro' ) && ! class_exists( 'ElementorPro\Plugin' ) ) {
```

Comment:
The purpose of this check is to make sure that you install Elementor Pro for local development too, since the main website requires it.  If this isn't what we want to do, then we can just remove this file instead.

Reply:
As example I don't need on translate website and this block create issue for me on getting ready the portal.
So in that way if the plugin is not available not warning me of that so I can keep warning and instead the others will have the error anyway.

Reply:
Ok, let's just remove it then.